test/e2e/go.mod

```diff
@@ -1,6 +1,8 @@
 module k8s.io/cloud-provider-vsphere/test/e2e
 
-go 1.20
+go 1.21
+
+toolchain go1.21.5
```

Comment:
This line is automatically generated if a lower version in go.mod than the expected toolchain, when running `go mod tidy` with golang 1.21 in the local.

Hi Yifeng, can you manually change line 3 from `go 1.20 `to `go 1.21` first and then run "go mod tidy"?  In this way we won't cause` toolchain go1.21.5` auto generated.

Reply:
Thanks you. Done.